imap_processing/codice/packet_definitions/P_COD_HI_INSTRUMENT_COUNTERS.xml

```diff
@@ -3,42 +3,42 @@
 	<xtce:Header date="2023-09" version="1.0" author="IMAP SDC" />
```

Comment:
These XML files should be in a different PR, they aren't related to the documentation.

Reply:
@maxinelasp how do I take these "out" of this commit?

Reply:
I know that we can do that through git reset soft command but I will let @maxinelasp or @greglucas chime in because I struggle with those kinds of things myself.

Reply:
I think you're still OK here Gabriel, but it won't be super easy/obvious how to update it either. I _think_ what happened is:
1. You updated the XML generator
2. You then ran the generator and updated the codice XML files
3. You committed **both** 1 and 2
4. You got feedback to remove 2 and tried to `git rm *.xml` in that directory
5. Now those files are _removed_ and thus have been updated in the opposite direction and still show changes

If that is right, then I think we want to just get back to point 1 above and commit that update?

The way I would approach that is by trying to get back to the original work you had and then modifying that commit. Here is a list of commands that I ran to accomplish this and push to my remote so you can see what it'd look like:
https://github.com/IMAP-Science-Operations-Center/imap_processing/compare/dev...greglucas:imap_processing:gml-xtce-update-docs

Note that during the rebase, I changed the first and third commit to "edit" from "pick" so that I could stop and modify those commits and remove the files from staging. I don't think that stuff is captured here, so these commands won't be verbatim what you need. Also notice the liberal use of `git status` and `git log` so I can see where I am in the process and what the current commit has in it.

```bash
10088  gh pr checkout 163
10089  git status
10090  git log
10091  git checkout -b gml-xtce-update-docs
10092  git fetch upstream
10093  git rebase -i upstream/dev
10094  git status
10095  git reset --soft HEAD~1
10096  git status
10097  git restore --staged imap_processing/codice/packet_definitions/*.xml
10098  git status
10099  git checkout imap_processing/codice/packet_definitions/*.xml
10100  git status
10101  git log
10102  git commit
10103  git rebase --continue
10104  git status
10105  git restore --staged imap_processing/codice/packet_definitions/*.xml
10106  git status
10107  git commit
10108  git status
10109  git rebase --continue
10110  git status
10111  git push -u origin gml-xtce-update-docs
10112  git log
```